packages/website/src/pages/index.tsx

```diff
@@ -16,7 +16,8 @@ interface FeatureItem {
 
 const features: FeatureItem[] = [
   {
-    title: 'What are ESLint and TypeScript, and how do they compare?',
+    title:
+      'Hello world: What are ESLint and TypeScript, and how do they compare?',
```

Comment:
@JamesHenry very odd, I saw the search on my local deployment from the commit in the PR:

![Screenshot of local deployment with search](https://user-images.githubusercontent.com/3335181/142770392-cd1da81a-5c13-43b0-a8d5-d08a151d7ec9.png)

Now, with this `Hello World:` change, I see it on the Netlify deployment too:

![Screenshot of Netlify deployment with Hello World and search](https://user-images.githubusercontent.com/3335181/142770864-d0b549d9-841f-4e65-a124-a859f20b328e.png)

🤷

Reply:
Might be related to caching... I can't pinpoint it either